app/src/main/res/layout/photo_view.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="8dp">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <ImageView
+            android:id="@+id/item_image"
+            android:layout_width="600dp"
+            android:layout_height="600dp"
```

Comment:
This is ok, and will work in 99% of cases. But Android peculiarity is that we have millions of different devices with different screen sizes (in contrast to ios where they have 5-6 devices). 
While 600dp works on usual phones it might be too small on tablets or too big on old/small devices or watches. **So the best way is to always avoid setting sizes in `dp` where we can.** It is not always possible but it's good to avoid it.

Let's try to use a ConstraintLayout here, make width and height `0dp` and make the image sqare by setting `app:layout_constraintDimensionRatio=1:1`.